pkg/collector/corechecks/snmp/checkconfig/config.go

```diff
@@ -0,0 +1,482 @@
+package checkconfig
+
+import (
+	"fmt"
+	"path/filepath"
+	"sort"
+	"strings"
+	"time"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/DataDog/datadog-agent/pkg/autodiscovery/integration"
+	"github.com/DataDog/datadog-agent/pkg/collector/check/defaults"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+
+	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/snmp/common"
+	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/snmp/metadata"
+)
+
+// Using high oid batch size might lead to snmp calls timing out.
+// For some devices, the default oid_batch_size of 5 might be high (leads to timeouts),
+// and require manual setting oid_batch_size to a lower value.
+var defaultOidBatchSize = 5
+
+var defaultPort = uint16(161)
+var defaultRetries = 3
+var defaultTimeout = 2
+var subnetTagPrefix = "autodiscovery_subnet"
+
+// DefaultBulkMaxRepetitions is the default max rep
```

Comment:
I think it's the same topic as https://github.com/DataDog/datadog-agent/pull/8971#discussion_r706147007

Reply:
Not really, here it could help to have more details in the comment about what max repetition is. For example `The max-repetitions parameter indicates how many rows of the table are to be retrieved in a single GetBulk operation` (from https://www.ibm.com/docs/en/networkmanager/4.2.0?topic=v3-configuring-maximum-number-repetitions-getbulk-requests)

Reply:
Sure, added a comment here 38c226272